packages/astro/src/core/errors/errors-data.ts

```diff
@@ -791,6 +791,30 @@ export const InvalidDynamicRoute = {
 	message: (route: string, invalidParam: string, received: string) =>
 		`The ${invalidParam} param for route ${route} is invalid. Received **${received}**.`,
 } satisfies ErrorData;
+/**
+ * @docs
+ * @see
+ * - [Images](https://docs.astro.build/en/guides/images/)
+ * @description
+ * When using a [strict package manager](https://pnpm.io/pnpm-vs-npm#npms-flat-tree) like pnpm, Sharp must be installed manually into your project in order to use image processing.
+ *
+ * If you are not using image processing, you can use a passthrough image service that does no processing, like such:
```

Comment:
```suggestion
 * If you are not using `astro:assets` for image processing, and do not wish to install Sharp, you can configure the following passthrough image service that does no processing:
```
I think saying `astro:assets` here can be supported by the rest of docs where we can make the distinction of image processing vs not by saying whether `astro:assets` is involved.